apps/prairielearn/src/pages/studentAssessmentInstance/studentAssessmentInstance.html.ts

```diff
@@ -0,0 +1,751 @@
+import { EncodedData } from '@prairielearn/browser-utils';
+import { html, unsafeHtml } from '@prairielearn/html';
+import { renderEjs } from '@prairielearn/html-ejs';
+
+import { Modal } from '../../components/Modal.html.js';
+import { StudentAccessRulesPopover } from '../../components/StudentAccessRulesPopover.html.js';
+import { compiledScriptTag } from '../../lib/assets.js';
+import { formatPoints } from '../../lib/format.js';
+
+export function StudentAssessmentInstance({ resLocals }: { resLocals: Record<string, any> }) {
+  return html`
+    <!doctype html>
+    <html lang="en">
+      <head>
+        ${renderEjs(import.meta.url, "<%- include('../partials/head'); %>", resLocals)}
+        ${resLocals.assessment.type === 'Exam'
+          ? html`${compiledScriptTag('examTimeLimitCountdown.ts')}
```

Comment:
I don't mind if this is moved to a later PR, would would it be better to add a conditional import in `studentInstanceQuestionClient` to this file instead? Just raising the question, feel free to ignore it.

Reply:
I'm inclined to ignore this for now.